aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormField, TextArea, RadioButtonGroup, FileInput } from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  let Input;
+  if (kind === 'star') Input = StarRating;
+  else if (kind === 'thumbs') Input = ThumbsRating;
+  else if (kind === 'textArea') Input = TextArea;
+  else if (kind === 'radioButtonGroup') Input = RadioButtonGroup;
+  else if (kind === 'fileInput') Input = FileInput;
+
+  if (kind === 'star' || kind === 'thumbs')
+    return (
+      <FormField
+        contentProps={{
+          border: false,
+        }}
+        label={label}
+        name={name}
+        htmlFor={name}
+      >
+        <Input {...accessibilityProps} {...inputProps} />
+      </FormField>
+    );
+
+  return (
+    <FormField label={label} name={name} htmlFor={name}>
```

Comment:
The question text is feeling small and difficult to read. I'm thinking that the feedback context is different than a true form context, and the text is having to do a lot more communicating and heavy lifting.

```suggestion
      <FormField label={<Text>{label}</Text>} name={name} htmlFor={name}>
```

Reply:
are you referring to just the first question with the thumbs up and thumbs down?

Reply:
![Screen Shot 2022-10-03 at 3 13 31 PM](https://user-images.githubusercontent.com/1756948/193690880-74c351b4-cb31-4b51-8d0f-7ea6f2827daf.png)